inc/template-tags.php

```diff
@@ -313,7 +313,7 @@ function _s_display_copyright_text() {
 	}
 
 	?>
-	<span class="copyright-text"><?php echo wp_kses_post( $copyright_text ); ?></span>
+	<span class="copyright-text"><?php echo wp_kses_post( do_shortcode( $copyright_text ) ); ?></span>
```

Comment:
This does not appear to work as intended; if I use the shortcode in the customizer field for Copyright Text:
![](https://dl.dropbox.com/s/p0ebr4u6g07olen/Screenshot%202018-09-06%2013.52.04.jpg?dl=0)

The actual shortcode is output on the frontend:
![](https://dl.dropbox.com/s/ob50cj66datsy7i/Screenshot%202018-09-06%2013.53.12.jpg?dl=0)